lib/web/resources_test.go

```diff
@@ -375,7 +375,6 @@ func TestListResources(t *testing.T) {
 			expected: proto.ListResourcesRequest{
 				ResourceType:        types.KindNode,
 				Limit:               defaults.MaxIterationLimit,
-				NeedTotalCount:      true,
```

Comment:
Why are we removing this field?

Reply:
so before, i set this field to always be true because i thought the list resources request coming through the web api layer will always require it (bad decision on my part), but this field isn't actually necessary because our frontend calls with the query `sort` always set, which will also return the total count.